app/build.gradle

```diff
@@ -1,88 +1,84 @@
+
+    defaultConfig {
+        applicationId "io.bloco.template"
+        minSdkVersion 21
+        targetSdkVersion 30
+        versionCode 1
+        versionName "1.0"
+
+        testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
-  }
-
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_8
-    targetCompatibility JavaVersion.VERSION_1_8
-  }
-
-  // Share code between test environments
-  sourceSets {
-    String sharedTestDir = 'src/sharedTest/java'
-    test {
-      java.srcDir sharedTestDir
+
+    kapt {
+        generateStubs = true
     }
-    androidTest {
-      java.srcDir sharedTestDir
+
+    buildTypes {
+        release {
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+        }
     }
-  }
 }
 
 dependencies {
-  compile fileTree(include: ['*.jar'], dir: 'libs')
-  compile 'com.android.support:appcompat-v7:25.3.1'
-  compile 'com.android.support:design:25.3.1'
-  // Architecture Components
-  compile 'android.arch.lifecycle:runtime:1.0.0-alpha3'
-  compile 'android.arch.lifecycle:extensions:1.0.0-alpha3'
-  annotationProcessor 'android.arch.lifecycle:compiler:1.0.0-alpha3'
-  // Dagger
-  annotationProcessor 'com.google.dagger:dagger-compiler:2.11'
-  compile 'com.google.dagger:dagger:2.11'
-  provided 'javax.annotation:jsr250-api:1.0'
-  // Logging
-  compile 'com.jakewharton.timber:timber:4.5.1'
-  // View injection
-  compile 'com.jakewharton:butterknife:8.6.0'
-  annotationProcessor 'com.jakewharton:butterknife-compiler:8.6.0'
-  // Reactive programming
-  compile 'io.reactivex.rxjava2:rxandroid:2.0.1'
-  compile 'io.reactivex.rxjava2:rxjava:2.1.0'
-  // Reactive lifecycle
-  compile 'com.trello.rxlifecycle2:rxlifecycle:2.1.0'
-  compile 'com.trello.rxlifecycle2:rxlifecycle-android:2.1.0'
-  compile 'com.trello.rxlifecycle2:rxlifecycle-android-lifecycle:2.1.0'
-  // Testing
-  testCompile 'junit:junit:4.12'
-  testCompile 'com.google.truth:truth:0.32'
-  testCompile 'com.github.blocoio:faker:1.2.5'
-  testCompile 'com.android.support:appcompat-v7:25.3.1'
-  testCompile 'com.android.support:design:25.3.1'
-  testCompile 'org.mockito:mockito-core:2.8.9'
-  androidTestCompile 'junit:junit:4.12'
-  testCompile 'com.google.truth:truth:0.32'
-  androidTestCompile 'com.github.blocoio:faker:1.2.5'
-  androidTestCompile 'com.android.support:appcompat-v7:25.3.1'
-  androidTestCompile 'com.android.support:design:25.3.1'
-  androidTestCompile 'com.android.support:support-annotations:25.3.1'
-  androidTestCompile 'com.android.support.test:runner:0.5'
-  androidTestCompile 'com.android.support.test:rules:0.5'
-  androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.2'
-  androidTestCompile('com.android.support.test.espresso:espresso-contrib:2.2.2') {
-    exclude module: 'recyclerview-v7'
-    exclude module: 'support-v4'
-  }
+    implementation fileTree(dir: "libs", include: ["*.jar"])
```

Comment:
I usually take this line out, never used a `jar` library file in the last 3 years.